src/org/labkey/test/components/glassLibrary/grids/DetailTableEdit.java

```diff
@@ -12,15 +16,16 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class DetailTableEdit extends WebDriverComponent
+public class DetailTableEdit extends WebDriverComponent<DetailTableEdit.ElementCache>
 {
-
-    private final WebElement _editPanel;
+    private final WebElement _formElement;
     private final WebDriver _driver;
+    private String _sourceTitle;
```

Comment:
This variable doesn't make sense to me. There is a setter for it:
`public DetailTableEdit setSourcePanelTitle(String sourcePanelTitle)`
The getter:`private String getSourceTitle()` uses the variable more as a flag (checks for null value) and if it is not null the getter ignores whatever value of the variable and uses the value from the panel header.
And since getter is private this a write-only variable, which is odd.

It feel likes you can get rid of this variable and change getSourceTitle to:
`    private String getSourceTitle()
    {
          return getTitle().replace("Editing ", "");
    }
`

Reply:
The intention for its use is to find the original table after finishing the edit.
Probably it would be best to pass it as a constructor param instead